lib/stencil-push.spec.js

```diff
@@ -0,0 +1,94 @@
+'use strict';
+
+const Code = require('code');
+const Lab = require('lab');
+const lab = exports.lab = Lab.script();
+const describe = lab.describe;
+const expect = Code.expect;
+const it = lab.it;
+const rewire = require("rewire");
+const sinon = require('sinon');
+const StencilPush = rewire('./stencil-push');
+const Wreck = require('wreck');
+const mockDb = {
+    _data: {},
+    get data() {
+        return this._data;
+    },
+    set data(data) {
+        this._data = data;
+    },
+}
+
+describe('stencil push', () => {
+    lab.beforeEach(done => {
+        mockDb.data = {};
+
+        this.wreckGetStub = sinon.stub(Wreck, 'get').callsFake(requestStub);
+        this.wreckPostStub = sinon.stub(Wreck, 'post').callsFake(requestStub);
+        this.wreckPutStub = sinon.stub(Wreck, 'put').callsFake(requestStub);
+
+        this.restoreStencilPush = StencilPush.__set__({
+            "utils.generateBundle": utilStub({
+                bundleZipPath: 'bundleZipPath',
+            }),
+            "utils.promptUserWhetherToApplyTheme": utilStub({
+                applyTheme: true,
+            }),
+            "utils.promptUserForVariation": utilStub({
+                variationId: 'bold',
+            }),
+        });
```

Comment:
You don't need rewire to stub this. 
You can stub `utils` directly instead, something like this:

```js
const utils = require('./stencil-push.utils.js');
sinon.stub(utils, 'generateBundle');
```